### PR TITLE
Add `tubingen_apg` profile

### PR DESCRIPTION
**Pull request by TCLamnidis**
---
name: tubingen_apg
about: A new cluster config
---

Please follow these steps before submitting your PR:

- [x] If your PR is a work in progress, include `[WIP]` in its title
- [x] Your PR targets the `master` branch
- [x] You've included links to relevant issues, if any

Steps for adding a new config profile:

- [x] Add your custom config file to the `conf/` directory
- [x] Add your documentation file to the `docs/` directory
- [x] Add your custom profile to the `nfcore_custom.config` file in the top-level directory
- [x] Add your custom profile to the `README.md` file in the top-level directory
- [x] Add your profile name to the `profile:` scope in `.github/workflows/main.yml`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -155,6 +155,7 @@ Currently documentation is available for the following systems:
 - [SEG_GLOBE](docs/seg_globe.md)
 - [Super Computing Wales](docs/scw.md)
 - [TIGEM](docs/tigem.md)
+- [University of Tuebingen, APG Group](docs/tubingen_apg.md)
```
Comment (jfy133) [inline README.md:158]:
```suggestion
- [TUBINGEN_APG](docs/tubingen_apg.md)
```

File: docs/tubingen_apg.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+```
+/opt/images-live/$USER/nf-core
+```
+
+The following directories are bound to the container on execution:
+- `/nfs/geo`
+- `/home`
+- `/local_scratch`
+- `/opt/reference`
+- `/opt/resources`
```
Comment (jfy133) [inline docs/tubingen_apg.md:20]:
```suggestion
Using this profile, a docker image containing all of the required software will be downloaded, and converted to a singularity container before execution of the pipeline. The image will be stored in the cache directory:

```
/opt/container_tmp/apptainer/nextflow
```
```

**Review by jfy133 — CHANGES_REQUESTED** [review]
Mostly docs stuff but otherwise loks good

File: docs/tubingen_apg.md (hunk 2 above)
Comment (TCLamnidis) [inline docs/tubingen_apg.md]:
```suggestion
Using this profile, a docker image containing all of the required software will be downloaded, and converted to a singularity container before execution of the pipeline. The image will be stored in the cache directory:
`/opt/container_tmp/apptainer/nextflow`
```

File: docs/tubingen_apg.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+This configuration will automatically choose the correct SLURM queue (short,medium,long) depending on the time and memory required by each process.
+
+Using this profile, a docker image containing all of the required software will be downloaded, and converted to a singularity container before execution of the pipeline. The image will be stored in the cache directory:
+
+```
+/opt/container_tmp/apptainer/nextflow
+```
+
+> NB: Nextflow will need to submit the jobs via SLURM to the clusters and as such the commands above will have to be executed on one of the head nodes. If in doubt contact IT.
```
Comment (scarlhoff) [inline docs/tubingen_apg.md:15]:
```suggestion
> NB: The commands to run a pipeline will have to be executed on one of the head nodes, so Nextflow can submit the jobs to the clusters via SLURM. If in doubt, contact IT.
```

**Review by jfy133 — APPROVED** [review]

**Comment by jfy133** [comment]
I agree with @scarlhoff it _should_ be Tuebingen, but technically and documentally everything is OK to me now.

**Review by scarlhoff — APPROVED** [review]

**Pull request merged**